students/KellyM/Lesson2/lesson 2 - series.py

```diff
@@ -0,0 +1,48 @@
+def fib(n):
+    """ This function calculates a Fibonacci Series, which is a series that
+        starts with 0 and 1, then calculates the next number by adding the
+        previous two numbers."""
+    if n == 1:
+        return 1
+    elif n==2:
+        return 1
+    else:
+        return (fib(n-1) + fib(n-2))
+
+def lucas(n):
+    """This function calculates Lucas Numbers, which are a series that
+        starts with 2 and 1, then calculates the next number by adding the
+        previous two numbers."""
+    if n == 0:
+        return 2
+    elif n==1:
+        return 1
+    else:
+        return (lucas(n-1) + lucas(n-2))
+
+def sum_series(n, o=0, p=1):
+    """This function calculates either the Fibonacci Series or the Lucas
+        Numbers, depending on if additional parameters are provided. If no
+        optional parameters, the Lucas Numbers will be calculatted,
+        otherwise, the Fibonacci Series will be caluslated."""
+    if o or p is None:
+        return fib(n)
+    else:
+        return lucas(n)
+
```

Comment:
The goal here is to create a series that defaults to fibonacci but can take any two numbers as the
first two values to produce a new series of numbers. You've got the right idea in setting the keyword arguments to 0 and 1 to produce the fibonacci series, but your function should work in such a way that the lucas series
would be produced by calling with `o=2` and `p=1` as the keyword arguments, so that `sum_series(7, 2, 1) == 29`. Other series could then be produced by substituting new values for the optional keyword arguments